Formula/c/crashpad.rb

```diff
@@ -0,0 +1,181 @@
+class Crashpad < Formula
+  desc "Crash reporting system for client applications"
+  homepage "https://chromium.googlesource.com/crashpad/crashpad"
+  url "https://chromium.googlesource.com/crashpad/crashpad/+archive/8da335ffad361191fa467de56a30715223f04014.tar.gz"
+  version "8da335ffad361191fa467de56a30715223f04014"
```

Comment:
This does not seem to be a stable version, but instead just a random commit: https://docs.brew.sh/Acceptable-Formulae#stable-versions

Reply:
I checked for version on their GitHub (https://github.com/chromium/crashpad), but it's not available there. I even scoured the repository for the same, and checked the commit history but no clue

Reply:
@SMillerDev Crashpad doesn't have any versioning strategy, actually. It comes from Chromium, where they just consume the tip of it. The only closest versioning strategy I can think of is checking what version a specific Chromium release uses and use that?

Reply:
Chromium is currently using the same revision as mentioned in the PR

Ref: https://source.chromium.org/chromium/chromium/src/+/main:third_party/crashpad/README.chromium

Reply:
> Crashpad doesn't have any versioning strategy, actually. It comes from Chromium, where they just consume the tip of it.

Then it's not acceptable for Homebrew/core. 
Please consider hosting it in a personal tap, which is very easy to do: https://docs.brew.sh/How-to-Create-and-Maintain-a-Tap

Reply:
Oh okay, thank for sharing the information!

Reply:
@SMillerDev Would it be OK to use the corresponding Chromium versions instead, as they are the versions that govern all the dependencies as a mono repo? For example, if Chromium v1.2.3 uses a certain SHA of Crashpad, then set the Crashpad formula to v1.2.3? 

Alternatively, seems like Crashpad on VCPKG uses a timestamp instead: https://vcpkg.roundtrip.dev/ports/crashpad. 

Crashpad and its utilities are used by lots of people out there, so having it as an official formula would be great!

Reply:
It's not really a stable version from upstream though. The best way would really be for upstream to publish tags.

Reply:
If start maintaining a fork of crashpad and add version tags to it myself, would it be acceptable to create a formula for that?

Reply:
I had posted about this on the crashpad-dev google group and I have received a response from the Crashpad maintainers - https://groups.google.com/a/chromium.org/g/crashpad-dev/c/YABNQm6Hcss

Reply:
> If I start maintaining a fork of crashpad and add version tags to it myself, would it be acceptable to create a formula for that?

@Moisan, could you help us with this?

Reply:
> If I start maintaining a fork of crashpad and add version tags to it myself, would it be acceptable to create a formula for that?

Seeing how upstream says that's a bad idea, and we specifically don't want to package forks.. that's not a solution